tests/Integration/Service.Tests/StockTriggerLevelsModuleSpecs/WhenSearchingPalletNumbers.cs

```diff
@@ -16,7 +16,7 @@
 
     using NUnit.Framework;
 
-    public class WhenSearching : ContextBase
+    public class WhenSearchingPalletNumbers : ContextBase
```

Comment:
this test looks functionally identical to the one above.. whats the need for both?

Reply:
both tests are just binding 'storagePlaceSearchTerm' from the request and everything after that point is mocked out anyway, so i think you can just delete one of them

Reply:
Yeah, fair point. Will do.